docs/UserGuide.md

```diff
@@ -275,14 +278,13 @@ Format: `sort COMPARISON_MEANS`
 
 * The valid options for the sorting method `COMPARISON_MEANS` are `name`, `classTime` or `year`.
 * Only one option for the sorting method can be specified.
-* The sorting method is case sensitive when being specified
-* Sorting methods:
+* Sorting methods (case sensitive):
```

Comment:
Hi! I would be making changes to my section of the user guide separately so there's no need to update this part for now. Also, for line 281 the "(case sensitive)" portion, I think it can be confusing so I am thinking it would be better on another line

Reply:
Alright, that should do it